src/components/composer/composer/composer.ts

```diff
@@ -268,37 +262,32 @@ export class Composer extends Component<ComposerProps, SpreadsheetChildEnv> {
         this.autoComplete(autoCompleteValue);
         return;
       }
-    } else {
-      // when completing with tab, if there is no value to complete, the active cell will be moved to the right.
-      // we can't let the model think that it is for a ref selection.
-      // todo: check if this can be removed someday
-      this.env.model.dispatch("STOP_COMPOSER_RANGE_SELECTION");
     }
 
     const direction = ev.shiftKey ? "left" : "right";
     this.env.model.dispatch("STOP_EDITION");
     this.env.model.selection.moveAnchorCell(direction, 1);
   }
 
+  processSpaceKey(ev: KeyboardEvent) {
+    if (ev.ctrlKey) {
```

Comment:
Not rellay the scope of this PR, but functionnaly shouldn't ctrl+enter be a toggle between range selection/formula ?

Reply:
as far as I know, the shortcut for this is F2, which we still ahven't implemented. where have you see nCtrl+Enter ?

Reply:
ctrl+space* I meant. which is what the method does.

Reply:
ok ^^ well my question stands, I don't see that behaviour in GSheet for instance

Reply:
Well yeah I don't see it either, but the fact is taht we implemented a ctrl+space shortcut here